Tools/Scripts/libraries/webkitcorepy/webkitcorepy/autoinstall.py

```diff
@@ -433,6 +429,11 @@ class AutoInstall(object):
     DISABLE_ENV_VAR = 'DISABLE_WEBKITCOREPY_AUTOINSTALLER'
     CA_CERT_PATH_ENV_VAR = 'AUTOINSTALL_CA_CERT_PATH'
 
+    # This list of libraries is required to install other libraries, and must be installed first
+    BASE_LIBRARIES = ['setuptools', 'wheel', 'pyparsing', 'packaging', 'setuptools_scm']
+    if sys.version_info >= (3, 0):
+        BASE_LIBRARIES.insert(-1, 'tomli')
```

Comment:
Since (IIUC) tomli is needed to install setuptools_scm, I think it's clearer and less error-prone to say:

```suggestion
    BASE_LIBRARIES = ['setuptools', 'wheel', 'pyparsing', 'packaging']
    if sys.version_info >= (3, 0):
        BASE_LIBRARIES += ['tomli', 'setuptools_scm']
    else:
        BASE_LIBRARIES += ['setuptools_scm']
```

(And you could of course collapse this into one or two statements as desired)